common-compose.yml

```diff
@@ -0,0 +1,68 @@
+wordpress:
```

Comment:
We'll want to eliminate the `common-compose.yml` in favor of a `docker-compose.yml` that works on Triton and a `local-compose.yml` that extends the `docker-compose.ml` and works for local environments. See an examples in:

- https://github.com/autopilotpattern/jenkins/blob/master/docker-compose.yml
- https://github.com/autopilotpattern/jenkins/blob/master/local-compose.yml